doc/release/1.9.0-notes.rst

```diff
@@ -70,60 +145,806 @@ New features
+to use the approach of Wang and Zhang from their 2008 conference paper
+*Simpson’s rule based FFT method to compute densities of stable distribution*,
+making this method more competitive with the default method. In addition,
+users now have the option to change the parametrization of the Levy Stable
+distribution to Nolan's "S0" parametrization which is used internally by
+SciPy's pdf and cdf implementations. The "S0"  parametrization is described in
+Nolan's paper [*Numerical calculation of stable densities and distribution
+functions*](https://doi.org/10.1080/15326349708807450) upon which SciPy's
+implementation is based. "S0" has the advantage that ``delta`` and ``gamma``
+are proper location and scale parameters. With ``delta`` and ``gamma`` fixed,
+the location and scale of the resulting distribution remain unchanged as
+``alpha`` and ``beta`` change. This is not the case for the default "S1"
+parametrization. Finally, more options have been exposed to allow users to
+trade off between runtime and accuracy for both the default and FFT methods of
+pdf and cdf calculation. More information can be found in the documentation
+here (to be linked).
+
+- Added `scipy.stats.fit` for fitting discrete and continuous distributions to
+data.
+
+- The methods ``"pearson"`` and ``"tippet"`` from `scipy.stats.combine_pvalues`
+have been fixed to return the correct p-values, resolving
+[#15373](https://github.com/scipy/scipy/issues/15373). In addition, the
+documentation for `scipy.stats.combine_pvalues` has been expanded and improved.
+
+- Unlike other reduction functions, ``stats.mode`` didn't consume the axis
+being operated on and failed for negative axis inputs. Both the bugs have been
+fixed. Note that ``stats.mode`` will now consume the input axis and return an
+ndarray with the ``axis`` dimension removed.
+
+- Replaced implementation of `scipy.stats.ncf` with the implementation from
+Boost.
+
+- Add a `bits` parameter to `scipy.stats.qmc.Sobol`. It allows to use from 0
+to 64 bits to compute the sequence. Default is ``None`` which corresponds to
+30 for backward compatibility. Using a higher value allow to sample more
+points. Note: it does not correspond to the output type.
+
+- Add a `integers` method to `scipy.stats.qmc.QMCEngine`. It allows sampling
+integers using any QMC sampler.
+
+- improved the fit speed and accuracy of ``stats.pareto``
+
+- added ``qrvs`` method to ``NumericalInversePolynomial`` to match the
+situation for ``NumericalInverseHermite``
+
+- faster random variate generation for ``gennorm``
+
+- ``lloyd_centroidal_voronoi_tessellation`` has been added to allow improved
+sample distributions via iterative application of Voronoi diagrams and
+centering operations
 
 Other
 -----
 
+A draft developer CLI is available for SciPy, leveraging the ``doit``,
+``click`` and ``rich-click`` tools. For more details, see
+[gh-15959](https://github.com/scipy/scipy/pull/15959).
```

Comment:
Maybe we can fuse the two "Other" themed sections?

Reply:
Yes, good idea.